distrobox-init

```diff
@@ -275,6 +275,21 @@ if ! command -v mount || ! command -v mount || ! command -v passwd ||
 			shadow \
 			sudo \
 			util-linux
+	elif command -v emerge; then
+		# set the password requirements to the minimum. gentoos passwd policy is stricter than other distros
+		# otherwise setting up the container can fail.
+		# this is needed on ubuntu 21.10
+		# works by setting /etc/security/passwdqc.conf to min=1,1,1,1,1 + everything except the first line from
+		# the original
+		printf "min=1,1,1,1,1\n%s" "$(tail -n +2 /etc/security/passwdqc.conf)" > /etc/security/passwdqc.conf
```

Comment:
There are a couple of whitespaces problems, run a `shfmt -s` on the file and it should be fixed
currently the linting github actions fails for this

Reply:
Yep